README.md

```diff
@@ -36,46 +29,47 @@ https://terraform-ibm-modules.github.io/documentation/#/implementation-guideline
 
 ### Usage
 
-<!--
-Add an example of the use of the module in the following code block.
-
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
-
 ```hcl
```

Comment:
You seem to be missing the module title heading after Overview but before Usage. See module template -> https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/619aa36ccb4860c97ced96735b6c33df144e22d3/README.md?plain=1#L40